pyuvdata/uvdata/uvfits.py

```diff
@@ -681,14 +561,31 @@ def read_uvfits(
                 # CASA misspells this one
                 self.timesys = ant_hdu.header["TIMSYS"]
 
+            prefer_lat_lon_alt = False
             if "FRAME" in ant_hdu.header.keys():
-                xyz_telescope_frame = ant_hdu.header["FRAME"]
+                if ant_hdu.header["FRAME"] == "ITRF":
+                    # uvfits uses ITRF, astropy uses itrs. They are the same.
+                    self._telescope_location.frame = "itrs"
+                elif ant_hdu.header["FRAME"] == "????":
+                    # default to itrs, but use the lat/lon/alt to set the location
+                    # if they are available.
+                    prefer_lat_lon_alt = True
+                    self._telescope_location.frame = "itrs"
```

Comment:
I think I would issue a warning here that a particular frame is being assumed, just so that the user is aware that it wasn't specified in the file.